templates/solutions/telco/5g-core.html

```diff
@@ -347,17 +347,17 @@ <h4 class="p-text--small-caps">What's included</h4>
               </div>
               <div class="col-4">
                 <ul class="p-list--divided">
-                  <li class="p-list__item is-ticked">Cloud native</li>
                   <li class="p-list__item is-ticked">Automated scaling</li>
                   <li class="p-list__item is-ticked">Self-healing</li>
                   <li class="p-list__item is-ticked">Rolling updates</li>
+                  <li class="p-list__item is-ticked">Enhanced Platform Awareness (EPA)</li>
                 </ul>
               </div>
             </div>
           </div>
           <hr class="p-rule--muted" />
           <p>
-            <a href="https://ubuntu.com/engage/hpe-telco-validated-design-with-canonical-kubernetes">Discover how HPE and Canonical's Kubernetes Telco validated design can future-proof your network&nbsp;&rsaquo;</a>
+            <a href="https://ubuntu.com/engage/hpe-telco-validated-design-with-canonical-kubernetes">Learn more about Canonical Kubernetes&nbsp;&rsaquo;</a>
```

Comment:
```suggestion
            <a href="https://ubuntu.com/kubernetes">Learn more about Canonical Kubernetes&nbsp;&rsaquo;</a>
```

Reply:
Ah, good catch, thanks. Didn't realise the href changed as well.